index.bs

```diff
@@ -220,20 +220,23 @@ is defined as follows:
 * If |N| is an {{Element}} which generates one or more <a>boxes</a>, the
     starting point of |N| in |C| is the two-dimensional offset in <a>pixel
     units</a> from the origin of |C| to the <a>flow-relative</a> starting corner
-    of the first <a>fragment</a> of the <a>principal box</a> of |N|, calculated
-    as if every <a>transformed element</a> had a <a>transformation matrix</a>
-    equal to the identity matrix.
+    of the first <a>fragment</a> of the <a>principal box</a> of |N|.
 
 * If |N| is a <a>text node</a>, the starting point of |N| in |C| is the
     two-dimensional offset in <a>pixel units</a> from the origin of C to the
     <a>flow-relative</a> starting corner of the first <a>line box</a> generated
-    by |N|, calculated as if every <a>transformed element</a> had a
-    <a>transformation matrix</a> equal to the identity matrix.
+    by |N|.
 
-NOTE: The CSS transform is ignored for the calculation of the starting point,
-to ensure that a node is not made unstable by a transform change. However, the CSS
-transform is not ignored for the calculation of the visual representation and
-the associated exclusion of points outside of the viewport.
+The <dfn export>transform-indifferent starting point</dfn> of a {{Node}} |N| in a
```

Comment:
Just a side note: we may later need a "scroll-indifferent starting point" concept for ignoring scroll shift, and the counterpart of it for layout shift countering scroll.

Reply:
I think "scroll indifference" is already captured by the consideration of ancestor scrolling-contents coordinate spaces in the definition of "unstable".

Reply:
Ack.